CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - `cropped(to:)` fixed size checking. [#575](https://github.com/SwifterSwift/SwifterSwift/pull/575) by [ilyahal](https://github.com/ilyahal)
 - **NSAttributedString** 
   - Fixed `attributes` property crash when the string is empty. [#617](https://github.com/SwifterSwift/SwifterSwift/pull/617) by [LucianoPAlmeida](https://github.com/LucianoPAlmeida).
+- **Date**
+  - Fixed `tomorrow` computed property to be calculated using Calendar.date(byAdding:to:) instead of date.addingTimeInterval(86400) [#642](https://github.com/SwifterSwift/SwifterSwift/pull/642) by [AlexeiGitH](https://github.com/AlexeiGitH).
```

Comment:
@AlexeiGitH this changelog you put in `Fixed` while the `yesterday` entry is in `Changed`. I think they both belong in `Fixed`. Can you please move it?

Reply:
@guykogus Just to confirm, do you want me to change #641, or this PR? The changelog entry for this one is already in `Fixed` section.

Reply:
#641 is already merged, so can you please do it here?